pkg/kubevirt/client.go

```diff
@@ -27,10 +28,22 @@ import (
 )
 
 const (
-	vmiSubresourceURL       = "/apis/subresources.kubevirt.io/%s/namespaces/%s/virtualmachineinstances/%s/%s"
-	annDefaultSnapshotClass = "snapshot.storage.kubernetes.io/is-default-class"
+	vmiSubresourceURL               = "/apis/subresources.kubevirt.io/%s/namespaces/%s/virtualmachineinstances/%s/%s"
+	annDefaultSnapshotClass         = "snapshot.storage.kubernetes.io/is-default-class"
+	InfraStorageClassNameParameter  = "infraStorageClassName"
+	InfraSnapshotClassNameParameter = "infraSnapshotClassName"
 )
 
+type InfraTenantStorageSnapshotMapping struct {
+	VolumeSnapshotClasses []InfraToTenantMapping `yaml:"volumeSnapshotClasses,omitempty"`
+	StorageClasses        []InfraToTenantMapping `yaml:"storageClasses"`
+}
+
+type InfraToTenantMapping struct {
+	Infra  string `yaml:"infra"`
+	Tenant string `yaml:"tenant"`
+}
```

Comment:
what's the purpose of the `yaml` fields here?

Reply:
I forgot to remove them, I was unmarshalling the fields directly from the config map before.